hugegraph-store/hg-store-grpc/src/main/proto/store_session.proto

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.apache.hugegraph.store.grpc.session";
+option java_outer_classname = "HgStoreSessionProto";
+
+import "store_common.proto";
+import "store_stream_meta.proto";
+
+service HgStoreSession {
+  rpc Get2(GetReq) returns (FeedbackRes) {}
+  rpc BatchGet2(BatchGetReq) returns (FeedbackRes) {}
+  rpc Batch(BatchReq) returns (FeedbackRes){}
+  rpc Table(TableReq) returns (FeedbackRes){};
+  rpc Graph(GraphReq) returns (FeedbackRes){};
```

Comment:
Table/Graph looks a little strange, where are these two interfaces used , and what content do they return?

Reply:
They are mainly used in `org.apache.hugegraph.store.node.grpc.HgStoreNodeService` (the related migration has not been completed yet, and `store-node` will be migrated after the migration of this pr, `store-core` and `store-rocksdb`). 

These two interfaces mainly define the CRUD operations related to table and graph. They will return true or false depending on whether these operations have failed or not.